v8/mastodoncomments/README.md

```diff
@@ -0,0 +1,40 @@
+Use [Mastodon](https://joinmastodon.org/) as a comment system for static site generator [Nikola](https://getnikola.com).
+
+Mastodon is a federated microblogging network, based on the [ActivityPub](https://www.w3.org/TR/activitypub/) protocol.
+Users can respond to anchor posts in the Fediverse.
+
+# How to configure
+The settings for Mastodon Comments can be passed to the plugin as follows:
+
+```python
+COMMENT_SYSTEM = "mastodon"
+COMMENT_SYSTEM_ID = "<YOUR-SITE_NAME>"
+GLOBAL_CONTEXT = {
+    ...
+    "mastodon_config": {
+        'host': 'nerdculture.de',
+        'account_id': '109270094542366763',
+        'account': 'fluchtkapsel'
+        }
+    }
+```
+See [this page](https://khendrikse.netlify.app/blog/find-your-mastodon-account-id/) on how to find your Mastodon account id.
+
+# How to install
+You need to copy the files `mastodon.js` and `mastodon.css` from `assets/files/{css,js}/` to `files/{css,js}/` respectively. Additionally, it requires DOMpurify which you can install by
```

Comment:
Please restructure this to use ``assets/js`` and ``assets/css`` in output, just like the rest of Nikola does.

Reply:
Sure. I took localsearch as a template and got the order of `files` and `assets` wrong :-D. Maybe you want to take a look at that, too, then, for the sake of standardization.